src/main/App.ts

```diff
@@ -60,7 +67,8 @@ export class App {
    * */
   async registerHandlersAndRoutes(
     app: Express,
-    redisClient: RedisClient
+    redisClient: RedisClient,
+    firebaseadmin: any
```

Comment:
```suggestion
    redisClient: RedisClient,
    firebaseAdmin: Auth
```

I think we can make this lower camel case and also make it typed

Reply:
I refactored that, thanks